internal/nonkube/command.go

```diff
@@ -0,0 +1,29 @@
+package nonkube
```

Comment:
Should this be place in a utils package as it is not specific to nonkube?

Reply:
The tarball is also generic, maybe they should both move.

I was a bit skeptical about a generic `internal/utils`, but I think it
makes sense to place them in a more generic location.

I will move them to `internal/utils` for now, unless someone has a
better suggestion.